From 634d877fb840ceb5eb7bf266e4b6e64477af4414 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Fri, 7 Feb 2020 14:34:50 -0800
Subject: [PATCH 631/830] ANDROID: gki_defconfig: Relocate INCREMENTAL_FS
X-NVConfidentiality: public

Ensure we build incrementalfs.ko as a module. As GKI is not ready for
module artifacts, move this to the board fragments.

Bug: 133435829
Change-Id: Ice50969b49b437606a0d9fd64844c8cfff05536e
Signed-off-by: Alistair Delva <adelva@google.com>
---
 arch/arm64/configs/db845c_gki.fragment | 1 +
 arch/arm64/configs/gki_defconfig       | 1 -
 arch/x86/configs/gki_defconfig         | 1 -
 cuttlefish.fragment                    | 1 +
 4 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
index d317188..3b249f6 100644
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -60,6 +60,7 @@ CONFIG_QCOM_QFPROM=m
 CONFIG_INTERCONNECT_QCOM=y
 CONFIG_INTERCONNECT_QCOM_OSM_L3=m
 CONFIG_INTERCONNECT_QCOM_SDM845=m
+CONFIG_INCREMENTAL_FS=m
 CONFIG_WLAN_VENDOR_ATH=y
 CONFIG_ATH10K_AHB=y
 CONFIG_ATH10K=m
diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index b4c6b5f..f76d3bc 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -390,7 +390,6 @@ CONFIG_QUOTA=y
 CONFIG_QFMT_V2=y
 CONFIG_FUSE_FS=y
 CONFIG_OVERLAY_FS=y
-CONFIG_INCREMENTAL_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 0003573..c8f2844 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -328,7 +328,6 @@ CONFIG_QUOTA=y
 CONFIG_QFMT_V2=y
 CONFIG_FUSE_FS=y
 CONFIG_OVERLAY_FS=y
-CONFIG_INCREMENTAL_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
diff --git a/cuttlefish.fragment b/cuttlefish.fragment
index dbcc370..bdff8fa 100644
--- a/cuttlefish.fragment
+++ b/cuttlefish.fragment
@@ -16,3 +16,4 @@ CONFIG_VIRTIO_PMEM=m
 CONFIG_VIRTIO_INPUT=m
 CONFIG_VIRTIO_MMIO=m
 CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y
+CONFIG_INCREMENTAL_FS=m
-- 
2.7.4

